.github/workflows/bot.yml

```diff
@@ -148,3 +148,100 @@ jobs:
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
           commit_message: "style: apply lint fixes"
+
+  Criterion_benchmarks:
+    name: Performance degradation
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'benchmark')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run json_like_bench
+        if: contains(github.event.pull_request.labels.*.name, 'benchmark')
+        uses: boa-dev/criterion-compare-action@v3.2.4
+        with:
+          benchName: json_like_bench
```

Comment:
do we need to specify different steps for every file with benchName? or we can wrap it in single step for the whole set?
criterion-compare states that benchName is optional

Reply:
If we wrap it in single step it shows (Unrecognized option: 'save-baseline') which on further solving(https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options) the issue i couldnt solve it but i deepdive into criterion-compare which uses (https://github.com/BurntSushi/critcmp/blob/master/README.md) to compare benchmarks and this critcmp is known to work with Criterion 0.3.3. and we are using 0.5.3 version 
These are my views feel free to correct me

Reply:
I agree with @meskill, I think we need a way to not have to update the bot.yml every time we add a new benchmark.